plugins/org.jkiss.dbeaver.ui.editors.image/src/org/jkiss/dbeaver/ui/data/managers/image/ImagePanelEditor.java

```diff
@@ -32,15 +32,19 @@
 import org.jkiss.dbeaver.ui.controls.imageview.SWTImageViewer;
 import org.jkiss.dbeaver.ui.controls.resultset.ResultSetPreferences;
 import org.jkiss.dbeaver.ui.data.IStreamValueEditor;
+import org.jkiss.dbeaver.ui.data.IStreamValueEditorPersistent;
 import org.jkiss.dbeaver.ui.data.IValueController;
 
 import java.io.IOException;
 import java.io.InputStream;
+import java.nio.file.Path;
 
 /**
 * ImagePanelEditor
 */
-public class ImagePanelEditor implements IStreamValueEditor<AbstractImageViewer> {
+public class ImagePanelEditor implements IStreamValueEditor<AbstractImageViewer>, IStreamValueEditorPersistent<AbstractImageViewer> {
```

Comment:
See comment in the `IStreamValueEditorPersistent.java`

```suggestion
public class ImagePanelEditor implements IStreamValueEditorPersistent<AbstractImageViewer> {
```